plugin.video.videodevil/resources/catcher.list

```diff
@@ -352,12 +352,15 @@ url=%s
 quality=standard
 ########################################################
 title=bitporno.sx
-target="file":"([^"]+)","label":"720p"
+target="file":"([^"]+)","label":"(720p|1080p HD)"

```

Comment:
swap these around, otherwise if there are both a 720p and a 1080p entry it will only match the 720p and we prefer it to match the higher quality 1080p entry instead